nixos/modules/virtualisation/containers.nix

```diff
@@ -564,6 +575,21 @@ in
               '';
             };
 
+            networkNamespace = mkOption {
+              type = types.nullOr types.str;
+              default = null;
+              example = "physical";
+              description = ''
+                The name of the network namespace that the container will be
+                started with. The namespace must have already been created
+                (e.g. with <literal>networking.namespaces</literal> or
```

Comment:
I think here (and elsewhere) you might want use the `<option>` tag, but I've seen `<literal>` used too, so it's pretty confusing.